bouncer/bouncer/settings.py

```diff
@@ -0,0 +1,121 @@
+"""
+Django settings for bouncer project.
+
+Generated by 'django-admin startproject' using Django 2.0.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.0/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+with open('../.secrets/secret-key') as f:
+    SECRET_KEY = f.read()
```

Comment:
It is best to `.trim()` the value of this variable, since the file might contain trailing whitespace and/or a new-line.

Reply:
I fixed it.